packages/web-components/src/search/components/search-result-overlay.ts

```diff
@@ -0,0 +1,127 @@
+import { LitElement, css, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { theme } from '../../theme';
+import { SearchResult } from '../search-overlay';
+
+export class ProductSearchBarResultOverlay extends LitElement {
+
+    @property()
+    setSearchTerm = (term: string) => {};
+
+    @property()
+    noResultsMessage: string | null = null;
+
+    @property({ type: Array })
+    results: SearchResult[] | null = null;
+
+    @property({ type: Number })
+    selectedIndex = -1;
+    
+    connectedCallback(): void {
+        super.connectedCallback();
+    }
+
+    render() {
+        return html`
+            <div class="rw-result-container">
+                ${(!this.results ||
+                this.results.length < 1) ? html`
+                    <div class="rw-no-results">${this.noResultsMessage ?? 'No search results to show'}</div>
+                ` : html`
+                <div>
+                    ${this.results.map((result, index) => {
+                        return html`
+                        <div ?selected=${index === this.selectedIndex} class="rw-selected-result">
+                        ${result.searchTermPrediction ?
+                            html`
+                                <div class="rw-prediction-item-container" @click=${() => this.setSearchTerm(result.searchTermPrediction!.term ?? '')}>
+                                    <h3 class="rw-prediction-item">
```

Comment:
Jeg synes dine predictions er meget store. Det ser sådan her ud hos mig:
![image](https://github.com/Relewise/relewise-ui-components/assets/2621842/d52d20d1-7649-476e-89b4-eef7a23974bf)

Reply:
![image](https://github.com/Relewise/relewise-ui-components/assets/141124692/c51206ef-93ac-482e-8d32-301f10a2019c)


Jeg har nu gjort dem lidt mindre! 🙂

Reply:
Du kan kigge på det loop icon der er ude til højre for at få en fornemmelse om space i mellem. 🙂